### DISCUSSION TITLE
Inquiry about how to set the default for Table columns

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4933] [Plugins]
I'd like to display the project start date and end date in the project list on the project menu.
Where and how can I modify the code to make Project start, Project end, Actions default in the Table columns as shown below?
![화면 캡처 2024-06-25 150031](https://github.com/kimai/kimai/assets/165639563/e369ff8d-6b8b-44f0-b601-0381def12b79)

**Comment by seleondevel** [comment]
In the next Kimai update version, I hope that the project start date and end date will also be reflected in the list in the project menu.
@kevinpapst I'd like to hear your ideas on this.

**Reply by kevinpapst** [reply]
They are available:
<img width="1294" alt="Bildschirmfoto 2024-06-26 um 09 37 39" src="https://github.com/kimai/kimai/assets/533162/45ccd57b-bf8d-490d-8bcb-f8ee751f0f05">

**Reply by seleondevel** [reply]
@kevinpapst  If I access the project menu for the first time after signing up, I would like to make the project start date and end date visible in the project list.

**Reply by seleondevel** [reply]
![화면 캡처 2024-06-25 150031](https://github.com/kimai/kimai/assets/165639563/64ef253c-b459-400f-bd39-e1fb13a9e953)
@kevinpapst 
You have added the project start date and end date to the items that appear in the project selection list in the Kimai setting.
However, even if a newly subscribed user accesses the Project menu for the first time, the Project Start-End Date entry does not appear in the list.

Can I ask you to check it?

**Reply by kevinpapst** [reply]
default columns cannot be changed

this settings is for dropdown boxes: https://www.kimai.org/documentation/configurations.html#project
